### DISCUSSION TITLE
v1.3.0.0

### DISCUSSION THREAD

**Discussion by xmine64** [MicaForEveryone/MicaForEveryone#223] [Announcements]
**Note**: This is the last release with .NET Core 3.1. We may have a release without UI to be able to target something else that's still supported. I may release v1.4 with mixed C#/C++ targeting .NET Framework. A version 2.0 is planned to implement "Mica For Everyone" in C++ and the work on it is already started by @dongle-the-gadget.

## What's Changed
* Add support for creating running as Administrator on startup (#38)
* Add support for hiding tray icon (#56, #135, #192)
* Improve rule matching
* Clean uninstallation when using Inno Setup
* Fix MSIX package only contains English language
* Fix here and there by @dongle-the-gadget 

### Translation
* Update Russian Translation by @Danik2343 in https://github.com/MicaForEveryone/MicaForEveryone/pull/162
* Update German translation by @uDEV2019 in https://github.com/MicaForEveryone/MicaForEveryone/pull/158
* Update Japanese translation by @maboroshin in https://github.com/MicaForEveryone/MicaForEveryone/pull/165
* Update Simplified Chinese translation by @FrzMtrsprt in https://github.com/MicaForEveryone/MicaForEveryone/pull/166
* Updated Korean translations by @RTFTR in https://github.com/MicaForEveryone/MicaForEveryone/pull/170
* Fixed mistranslated thing + Added the dialogs requested with translation by @RTFTR in https://github.com/MicaForEveryone/MicaForEveryone/pull/171
* Add Polish translation by @Piteriuz in https://github.com/MicaForEveryone/MicaForEveryone/pull/181
* Update Spanish translation by @xoascf in https://github.com/MicaForEveryone/MicaForEveryone/pull/187
* Add Portuguese Brazilian translation by @dgsmiley18 in https://github.com/MicaForEveryone/MicaForEveryone/pull/217
* Add Dutch translation by @toineenzo in https://github.com/MicaForEveryone/MicaForEveryone/pull/219
* Update Italian translation by @A-Loot in https://github.com/MicaForEveryone/MicaForEveryone/pull/221

### Build
* Update WinGet Releaser workflow by @sitiom in https://github.com/MicaForEveryone/MicaForEveryone/pull/139
* [Experiment] update get-version-action dependencies by @dongle-the-gadget in https://github.com/MicaForEveryone/MicaForEveryone/pull/214

### Dependencies
* build(deps): bump Microsoft.Extensions.DependencyInjection from 6.0.0 to 6.0.1 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/150
* build(deps): bump Microsoft.Bcl.AsyncInterfaces from 6.0.0 to 7.0.0 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/173
* build(deps): bump Microsoft.Extensions.DependencyInjection from 6.0.1 to 7.0.0 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/172
* build(deps): bump microsoft/setup-msbuild from 1.1 to 1.2 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/200
* build(deps): bump microsoft/setup-msbuild from 1.2 to 1.3 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/201

## New Contributors
* @FrzMtrsprt made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/166
* @Piteriuz made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/181
* @dgsmiley18 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/217
* @toineenzo made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/219
* @A-Loot made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/221

**Full Changelog**: https://github.com/MicaForEveryone/MicaForEveryone/compare/v1.2.0.2...v1.3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/MicaForEveryone/MicaForEveryone/releases/tag/v1.3.0.0'>v1.3.0.0</a>.</em>

**Comment by sitiom** [comment]
Winget Releaser workflow did not trigger for some reason. Undrafting the release should have worked.

**Reply by sitiom** [reply]
I have manually made a PR for now: https://github.com/microsoft/winget-pkgs/pull/93839

**Comment by Sweep29** [comment]
I don't have the option to change rules in the Backdrop Type and Corner Mode.
The only option is DEFAULT
Why is this happening ?

**Reply by FrzMtrsprt** [reply]
These options are not available if you're on Windows 10

**Reply by Sweep29** [reply]
Thanks. I had a feeling this was the issue.
Thanks for verifying it though.
